src/main/java/step3/App.java

```diff
@@ -0,0 +1,23 @@
+package step3;
+
+import step3.model.CarGroup;
+import step3.view.InputView;
+
+import java.util.Scanner;
+
+public class App {
+    public static final int CRITERION_NUMBER = 4;
+    public static final int RANGE = 9;
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        InputView.viewCarCount();
+        int carCount = scanner.nextInt();
+        InputView.viewCount();
+        int count = scanner.nextInt();
+
+        Game game = new Game(count);
+        game.run(new CarGroup(carCount));
```

Comment:
```suggestion
        Game.of(count, cars).run();
```
이런 형태로 표현할 수 있도록 작성해보세요!
run이 결과에 대한 output을 반환하고, 다시 이걸 result가 받아서 출력해주면 좋을 것 같습니다 ㅎㅎ